test/test_optim.py

```diff
@@ -1024,6 +1024,49 @@ def test_fused_matches_forloop(self, device, dtype, optim_info):
             self.skipTest("MPS supports only torch.float16 and torch.float32")
         self._test_derived_optimizers(device, dtype, optim_info, "fused")
 
+    @optims(
+        [optim for optim in optim_db if "fused" in optim.supported_impls],
+        dtypes=floating_types_and(
+            torch.bfloat16,
+            torch.float16,
```

Comment:
we probably do not need to test more than one dtype, no?

Reply:
agreed. we don't need low prec cases